src/Core/src/Handlers/Window/WindowHandler.Windows.cs

```diff
@@ -73,5 +92,23 @@ public static void MapMenuBar(IWindowHandler handler, IWindow view)
 				windowManager.SetMenuBar(mb.MenuBar);
 			}
 		}
+
+		public static void MapFlowDirection(IWindowHandler handler, IWindow view)
+		{
+			if (view.FlowDirection == FlowDirection.MatchParent)
+				return;
```

Comment:
What happens if I set it to say RTL and then back to MatchParent? Will it un-set somehow?

How do you tell the world to "use the default"? Just removing the RTL flag?